templates/infra/linodeFirewall.yaml

```diff
@@ -12,18 +12,43 @@ spec:
   inboundPolicy: DROP
   inboundRules:
     - action: ACCEPT
-      label: intra-cluster
+      label: intra-cluster-tcp
       ports: "1-65535"
       protocol: "TCP"
       addresses:
         ipv4:
           - "10.0.0.0/8"
+      description: accept all tcp traffic within the vpc
+    - action: ACCEPT
+      label: intra-cluster-udp
+      ports: "1-65535"
+      protocol: "UDP"
+      addresses:
+        ipv4:
+          - "10.0.0.0/8"
+      description: accept all udp traffic within the vpc
+    - action: ACCEPT
+      label: intra-cluster-icmp
+      protocol: "ICMP"
+      addresses:
+        ipv4:
+          - "10.0.0.0/8"
+      description: accept all icmp traffic within the vpc
     - action: ACCEPT
       addresses:
         ipv4:
           - 0.0.0.0/0
         ipv6:
           - ::/0
-      ports: "6443"
+      ports: 6443, 8132
```

Comment:
question: why do we need port 8132?

Reply:
for konnectivity: https://github.com/linode/cluster-api-provider-linode/blob/main/cloud/services/loadbalancers.go#L19-L22